backend/README.md

```diff
@@ -429,6 +429,41 @@ providers and guides.
 
 [Configuration reference](./docs/Config.md)
 
+## User import
```

Comment:
Would be good to link to this section from the table of contents.
IMO, would also be better to move this section above `API specification` and `Configuration reference`.

Reply:
I have put it into the additional Topics Section.